teps/0115-tekton-catalog-git-based-versioning.md

```diff
@@ -663,6 +678,9 @@ improve the scalability and performance of the Hub - most of this will be implem
 We could provide guidelines and recommendations for creating Catalogs e.g. deciding whether to group resources together
 instead of splitting them into separate Catalogs. We can explore this in future work. 
 
+### Catlin 
+Today, [catlin bump](https://github.com/tektoncd/catlin#bump) command only supports directory-based versioning catalog. We could further extend the [catlin bump](https://github.com/tektoncd/catlin#bump) command to support git-based versioning catalogs (where the command will create a new git tag based on the current latest version).
+
```

Comment:
would it also generate the release notes?

Reply:
Good point! Added 😄